src/textual/geometry.py

```diff
@@ -631,18 +628,58 @@ def css(self) -> str:
     def unpack(cls, pad: SpacingDimensions) -> Spacing:
         """Unpack padding specified in CSS style."""
         if isinstance(pad, int):
-            return cls(pad, pad, pad, pad)
+            return Spacing.all(pad)
         pad_len = len(pad)
         if pad_len == 1:
             _pad = pad[0]
-            return cls(_pad, _pad, _pad, _pad)
+            return Spacing.all(_pad)
         if pad_len == 2:
             pad_top, pad_right = cast(Tuple[int, int], pad)
             return cls(pad_top, pad_right, pad_top, pad_right)
         if pad_len == 4:
             top, right, bottom, left = cast(Tuple[int, int, int, int], pad)
             return cls(top, right, bottom, left)
-        raise ValueError(f"1, 2 or 4 integers required for spacing; {pad_len} given")
+        raise ValueError(
+            f"1, 2 or 4 integers required for spacing properties; {pad_len} given"
+        )
+
+    @classmethod
+    def vertical(cls, amount: int) -> Spacing:
```

Comment:
I found myself reaching for similar utility constructors in other frameworks quite frequently, so thought they'd be a nice addition.

Handy for readable code too: `Spacing.horizontal(2) + Spacing.vertical(4)` is quite a readable way of getting a spacing with 2 on the horizontal edges, and 4 on the vertical.

Reply:
I'd probably not use that example myself, feels like a roundabout way of doing `Spacing.unpack(4,2)` But I can't argue it is more readable!